virttest/utils_disk.py

```diff
@@ -92,8 +90,9 @@ def is_mount(src, dst=None, fstype=None, options=None, verbose=False, session=No
             mount_result = process.run(mount_check_cmd, shell=True).stdout_text
         if verbose:
             LOG.info("Output of findmnt: %s", mount_result)
-    except process.CmdError:
-        pass
+    except Exception as e:
```

Comment:
`session.cmd_output_safe` will not raise exception if the status is not zero, please use `session.cmd` instead. and here should be `process.CmdError, aexpect.exceptions.ShellCmdError`

Reply:
Oh yeah, forgot to update that, thanks!